src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -158,4 +161,8 @@ public Request getHighestPriorityFulfillableRequest() {
   public String getLoggedInUserId() {
     return loggedInUserId;
   }
+
+  public DateTime getSystemReturnDateTime() {
```

Comment:
Thinking about it, the check in isn't necessarily a return. In this context, it is when the `check in processed date time`.

Would it be easy to rename this field and property?

Reply:
Sure, rather easy. Done.